docs/install/kubernetes.md

```diff
@@ -94,7 +108,21 @@ curl -sSLo kubectl-storageos.tar.gz \
 > 💡 You can find binaries for different architectures and systems in [kubectl
 > plugin](https://github.com/storageos/kubectl-storageos/releases).
 
-### Install Ondat
+### Option A: Install Ondat (embedded etcd)
+
+```bash
+kubectl storageos install \
+    --include-etcd \
+    --etcd-tls-enabled \
+    --admin-username "myuser" \
+    --admin-password "my-password"
```

Comment:
Agree with Ferran, probably better to add the storageclass to use here and in the other references